src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceStatus.kt

```diff
@@ -54,18 +59,22 @@ open class InstanceStatus : InstanceTask() {
                             }.joinToString("<br>")
 
                             val packagesInstalled = if (available) {
-                                sync {
-                                    aem.packagesBuilt.map { task ->
-                                        sync {
+                                try {
```

Comment:
when instance is booting up, running `instanceStatus` could fail because of unresponding package manager, now it will simply print `unknown` at `installed packages` column